frontend/ui/uimanager.lua

```diff
@@ -5,12 +5,12 @@ This module manages widgets.
 local Device = require("device")
 local Event = require("ui/event")
 local Geom = require("ui/geometry")
+local Input = Device.input
```

Comment:
Oh, I meant before the functions in a file. The way I was thinking is

```
Class
function
_
Class.instance
file.function
```

It seems more logical visually to me that way, but what you propose is probably more logical overall in going from "big" to "small". Anyway, that's what I meant when I said that I've already been doing it that way.